bower.json

```diff
@@ -64,6 +64,7 @@
     }
   },
   "resolutions": {
-    "tableExport.jquery.plugin": "~1.6.8"
+    "tableExport.jquery.plugin": "~1.6.8",
+    "jspdf": "1.1.239 || 1.3.2"
```

Comment:
This appears to be a dependancy conflict WITHIN tableExport.jquery.plugin.  Don't know why, but this line seems to be a simple enough fix.

Reply:
I don't know bower but this `or` in libs version looks really scary. Could you explain why you need it in other words one more time?

Reply:
The or sign was added by bower, not myself. When you run `bower install` without this line, it asks you to choose an option to resolve the conflict, you can also choose to save the change to the file. If you do so, this is the line that bower itself adds to the file.

Reply:
I think it appears due to an OR in the dependancies of `tableExport.jquery.plugin`

Reply:
These dependancies are required by tableExport.jquery.plugin:
`    "jspdf": "1.1.239 || 1.3.2",
    "jspdf-autotable": "2.0.14 || 2.0.17"`

The package jspdf-autotable in turn specifies that jspdf is needed:
`    "jspdf": "~1.0.178"`

This two version requirements are mutally exclusive.  I selected to the more up-to-date version requirement of `1.1.239 || 1.3.2` so it has just saved this version string to our "resolutions".